hazelcast/src/main/java/com/hazelcast/internal/cluster/impl/ClusterServiceImpl.java

```diff
@@ -124,6 +124,7 @@
     private volatile Address masterAddress;
 
     private volatile String clusterId;
+    private boolean previouslyDataMember;
```

Comment:
as far as I see, we use this field only for printing a log line while converting back to data member. Can we remove this field and print the log outside?

Reply:
removed to cluster-merge-task